src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+  const calculator = {
+    operate(callback, number) {
+      if (typeof callback === 'function') {
+        result = callback(result, number);
+      }
+
+      return this;
+    },
+
+    getResult() {
+      return result;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
```

Comment:
The `add` method currently returns the sum of `a` and `b` without affecting the internal `result`. If the intention is to use these methods to update the `result`, consider modifying the `result` variable instead of returning the sum directly.